### PR TITLE
FEAT: Add event tracking for Matomo

### PR DESCRIPTION
**Pull request by cc7768**

### CODE REVIEW

**Comment by cc7768** [comment]
Waiting for the vercel deployment, but it seems to be working locally and I believe this is ready for review.

@daywiss @Gamaranto @Tulun

File: src/components/SendAction/SendAction.tsx [hunk 1]
```diff
@@ -123,6 +125,19 @@ const SendAction: React.FC = () => {
...
     }
     if (canSend) {
+      // Matomo track send transactions
+      trackEvent(
+        {
+          category: "send",
+          action: "bridge",
+          name: tokenInfo && tokenInfo.symbol.concat(
+            " ", sendState.currentlySelectedFromChain.chainId.toString().concat(
+              "->", sendState.currentlySelectedToChain.chainId.toString()
```
Comment (daywiss) [inline src/components/SendAction/SendAction.tsx:135]: if you plan to parse this programmatically, you may want a more standard way to serialize the data.  if json is easy to parse in python or w/e, you could use something like 
```js
  name: JSON.stringify({
    symbol: tokenInfo.symbol,
    from: sendState.currentlySelectedFromChain.chainId,
    to: sendState.currentlySelectedToChain.chainId
  })
```
which will output `name = "{symbol:'ETH',to:'1',from:'13'}"` which can be run through  a JSON parser to form dictionary

**Review by daywiss — APPROVED** [review]
looks good, just have suggestion for complex data

File: src/components/SendAction/SendAction.tsx (hunk 1 above)
Comment (cc7768) [reply src/components/SendAction/SendAction.tsx]: I like this a lot. It will definitely be easier to parse.

Updated.

**Review by Gamaranto — APPROVED** [review]
lgtm. 
We should double check if events the user switching `toChain` because they change `fromChain` are tracked (and so are false positives in a way) or not

**Pull request merged**